README.md

```diff
@@ -68,6 +68,16 @@
 
 You can then check the provisioning status of your cluster by running `oc describe cluster <cluster_name>`
 
+## Developing Cluster Operator Controllers Locally
+
+If you are actively working on controller code you can save some time by compiling and running locally:
+
+  * Run the deploy playbooks normally.
+  * `oc delete DeploymentConfig cluster-operator-controller-manager` - This will be recreated the next time you run the deploy playbooks.
+  * `make build`
```

Comment:
Not sure if it's worth mentioning, but on my Mac, I usually do `go install ./cmd/cluster-operator` to produce a darwin binary.